src/HIVCohortFilter/HIVCohortFilterCase.jsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import FileSaver from 'file-saver';
-import { arrangerGraphqlPath } from '../localconf';
 import { fetchWithCreds } from '../actions';
+import {
+  guppyGraphQLUrl,
+  arrangerGraphqlPath,
+  useGuppyForExplorer,
+} from '../configs';
 
 class HIVCohortFilterCase extends React.Component {
   // Base class for the 3 NDH cohort filter apps. Meant to facilitate code reuse
-  static performQuery(queryString) {
+  static performQuery(queryString, variableString) {
+    const graphqlUrl = useGuppyForExplorer ? guppyGraphQLUrl : arrangerGraphqlPath;
     return fetchWithCreds({
-      path: `${arrangerGraphqlPath}`,
+      path: `${graphqlUrl}`,
       body: JSON.stringify({
         query: queryString,
+        variables: JSON.parse(variableString),
```

Comment:
In the case where we use arranger, variableString will be undefined right? And I think that will lead to the error Uncaught SyntaxError: Unexpected token u in JSON at position 0

Reply:
nice catch! fixed in https://github.com/uc-cdis/data-portal/pull/558/commits/1dd79067b948a0cd77857bbc25b6548e37931ed9